core/src/main/java/com/flipkart/gjex/core/config/GrpcConfig.java

```diff
@@ -28,27 +31,6 @@ public class GrpcConfig {
     @JsonProperty("server.executorThreads")
     private int executorThreads = 0;
 
-    public int getPort() {
-        return port;
-    }
-
-    public void setPort(int port) {
-        this.port = port;
-    }
-
-    public int getMaxMessageSize() {
-        return maxMessageSize;
-    }
-
-    public void setMaxMessageSize(int maxMessageSize) {
-        this.maxMessageSize = maxMessageSize;
-    }
-
-    public int getExecutorThreads() {
-        return executorThreads;
-    }
-
-    public void setExecutorThreads(int executorThreads) {
-        this.executorThreads = executorThreads;
-    }
+    @JsonProperty("grpcFilterConfig")
+    private GrpcFilterConfig grpcFilterConfig;
```

Comment:
Same here, should be `filterConfig` simply

Reply:
updated